File: Java/dowhileloop/table.java
package Java.dowhileloop;

import java.util.Scanner;

public class table {
    public static void main(String[] args) {
        int num,i=1;
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter The Number Whose Table You Want ");
        num=sc.nextInt();
        do{
            System.out.println(num+" x "+i+" = "+num*i);
            i++;
        }while(i<=10);
        sc.close();
    }
}

File: C Programming/I TECH CLASS/for loop/table.c
#include <stdio.h>

int main() {
 
    int i,num;
    printf("Enter The Number Whose Table You Want\n");
    scanf("%d",&num);

    for(i=1;i<=10;i++)
    {
        printf("%d x %d = %d\n",i,num,i*num);
    }
 
return 0;
}